### PR TITLE
[Tracking] Updates gh-pages with Google Analytics

### PR DESCRIPTION
**Pull request by tynandebold**
@sampi @zdlm @tynandebold

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <!DOCTYPE html>
 <link rel="icon" href="data:;base64,iVBORw0KGgo="/>
-<meta http-equiv="refresh" content="0;URL=/v10/"/>
+<meta http-equiv="refresh" content="0;URL=/v11/"/>
```
Comment (sampi) [inline index.html]: Can you change this back to v10 by hand?

File: package.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
-  "version": "0.0.33",
+  "version": "0.0.34",
   "versions": [
-    "10.1.0",
+    "12.0.0-beta.0",
```
Comment (sampi) [inline package.json]: Can you revert the ordering so it's increasing in version numbers from the top?

Comment (tynandebold) [reply package.json]: Done.

File: index.html (hunk 1 above)
Comment (tynandebold) [reply index.html]: Fixed.

**Pull request merged**